fractal_server/app/runner/v1/__init__.py

```diff
@@ -91,22 +91,21 @@ async def submit_workflow(
             slurm backend.
     """
 
-    # Declare runner backend and set `process_workflow` function
     settings = Inject(get_settings)
     FRACTAL_RUNNER_BACKEND = settings.FRACTAL_RUNNER_BACKEND
-    if FRACTAL_RUNNER_BACKEND == "local":
-        process_workflow = local_process_workflow
-    elif FRACTAL_RUNNER_BACKEND == "slurm":
-        process_workflow = slurm_process_workflow
-    else:
-        raise RuntimeError(f"Invalid runner backend {FRACTAL_RUNNER_BACKEND=}")
-
     with next(DB.get_sync_db()) as db_sync:
 
         job: ApplyWorkflow = db_sync.get(ApplyWorkflow, job_id)
         if not job:
             raise ValueError(f"Cannot fetch job {job_id} from database")
 
+        if FRACTAL_RUNNER_BACKEND in ["local", "local_experimental"]:
```

Comment:
This won't work.
If we map the new runner onto the old one, what would happen when the user tries to shut down a v1 job?